tests/test_wmb.py

```diff
@@ -0,0 +1,151 @@
+import sys
```

Comment:
While it's nice that you started digging into the test coverage, that wasn't necessarily part of this task. We can keep it for now, but for the future, try to be wary of "scope creep".